src/AnnotationThread.js

```diff
@@ -467,16 +464,6 @@ class AnnotationThread extends EventEmitter {
         }
 
         this.element.removeEventListener('click', this.renderAnnotationPopover);
-        this.element.removeEventListener('blur', this.handleBlur);
-    }
-
-    /**
-     * Called when the annotation element loses focus
-     *
-     * @return {void}
-     */
-    handleBlur() {
```

Comment:
Does removing this cause any other side effects? I don't think I remember why this was added in the first place

Reply:
I added it originally to solve the problem of showing the point annotation when we clicked away from the popover, but it turns out it was conflicting with the case where when you click on the point annotation focus goes to the popover anyway, so it kept removing the flipped class